docs/site/Model.md

```diff
@@ -607,7 +607,7 @@ Here are general attributes for property definitions:
       <td>Yes</td>
       <td>String</td>
       <td>
-        Property type. Can be any type described in <a href="LoopBack-types.html">LoopBack types</a>.
+        Property type. Can be any type described in <a href="https://loopback.io/doc/en/lb4/LoopBack-types.html">LoopBack types</a>.
```

Comment:
use `LoopBack-types.md`?

Reply:
Yes please, we should use local (relative) paths, not absolute URLs. The trick is to use `.md` extension, that way Jekyll converts `LoopBack-types.md` to `/doc/en/lb4/LoopBack-types.html` link.

Reply:
I don't think HTML takes local paths. From the preview, it has `http://127.0.0.1:4001/doc/en/lb4/LoopBack-types.md#object-types`. cc @bajtos

Reply:
that's interesting. Looking at the [Server.md](https://raw.githubusercontent.com/strongloop/loopback-next/master/docs/site/Server.md), it is also referencing other md files.

Reply:
Talked to @agnes512, it seems like things won't get converted as expected because it's inside a html table.